client/layout/guided-tours/tours/editor-insert-menu-tour.js

```diff
@@ -36,8 +38,19 @@ class RepositioningStep extends Step {
 		clearInterval( this.interval );
 	}
 
+	render() {
+		if ( this.props.isEditorModeVisual ) {
+			return super.render();
+		}
+		return null;
+	}
+
 }
 
+const ConnectedStep = connect( state => ( {
+	isEditorModeVisual: 'html' !== getPreference( state, 'editor-mode' ),
```

Comment:
Ah yeah, the idea really was that, if anything will ever change on the editor front, I can be quite sure it won't be the HTML mode. 😄 

Anyway: your comment is correct and I'll fix it. :)